app/components/address-mapper.tsx

```diff
@@ -169,6 +178,17 @@ const AddressMapper: React.FC<AddressMapperProps> = ({
     }
   }, [softStoryData, tsunamiData, liquefactionData]);
 
+  useEffect(() => {
+    if (currentView === "") {
```

Comment:
in future PR, let's look into if we can replace this bit of code (the conditional) partially or completely https://react.dev/reference/react/useLayoutEffect

Reply:
never mind, this doesn't seem to have an effect on the hazard card display